.github/workflows/ci.yml

```diff
@@ -34,7 +41,14 @@ jobs:
     - run: make test
 
   qa:
-    runs-on: ubuntu-latest
+    strategy:
```

Comment:
@fbarotov we should remove this matrix since `qa` doesn't contain any OS specific checks.

Reply:
got it, thanks a lot.